examples/hubert/dataset/hubert_dataset.py

```diff
@@ -309,14 +310,20 @@ def _collate_audio_label(
         kernel_size = 25
         stride = 20
         sample_rate = 16  # 16 per millisecond
-        if waveform.shape[1] > audio_size:
-            diff = waveform.size(1) - audio_size
+        audio_start = 0
+        waveform = waveform[0]
+        if waveform.shape[0] > audio_size:
+            diff = waveform.size(0) - audio_size
             audio_start = torch.randint(diff, size=(1,)) if rand_crop else 0
```

Comment:
If `rand_crop` is falsy value, this does not change the value of `audio_start`, thus the computation of `diff` becomes redundant. I think updating the condition will express the logic better.

```python
if waveform.shape[1] > audio_size and rand_crop:
    diff = waveform.size(0) - audio_size
    audio_start = torch.randint(diff, size=(1,))
```